Cargo.lock

```diff
@@ -1540,27 +1553,33 @@ dependencies = [
 
 [[package]]
 name = "deno_doc"
-version = "0.156.0"
+version = "0.159.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "2585b98d6ad76dae30bf2d7b6d71b8363cae041158b8780d14a2f4fe17590a61"
+checksum = "3522d34540ce1053fc35d61776a090b7007f76d0a9a5222579718cce28ec800d"
 dependencies = [
+ "ammonia",
  "anyhow",
  "cfg-if",
  "comrak",
  "deno_ast",
  "deno_graph",
+ "deno_path_util",
  "futures",
  "handlebars",
  "html-escape",
  "import_map",
  "indexmap 2.3.0",
  "itoa",
+ "js-sys",
  "lazy_static",
+ "percent-encoding",
  "regex",
  "serde",
+ "serde-wasm-bindgen",
```

Comment:
Probably the wasm build of deno_doc should be separated into a different crate so that serde-wasm-bindgen doesn't confusingly show up here. We've done it in the other crates and it's a bit cleaner.